salt/modules/mac_timezone.py

```diff
@@ -53,7 +53,7 @@ def _get_date_time_format(dt_string):
         except ValueError:
             continue
     msg = 'Invalid Date/Time Format: {0}'.format(dt_string)
-    raise CommandExecutionError(msg)
+    raise SaltInvocationError(msg)
```

Comment:
If we're going to raise exceptions here then we need to go back anything that calls this and ensure that we have documented that a caller can except them on an error.